ports/dxut/vcpkg.json

```
@@ -1,8 +1,8 @@
 {
   "name": "dxut",
-  "version-string": "11.14",
-  "port-version": 4,
+  "version-string": "11.25",
+  "port-version": 1,
```

```suggestion
  "port-version": 0,
```
Since this is new version update, it should be `0`.